Vegeting/Screens/Main/ViewController/MainTabBarViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MainTabBarViewController.swift
+//  Vegeting
+//
+//  Created by 김원희 on 2022/10/26.
+//
+
+import UIKit
+
+class MainTabBarViewController: UITabBarController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
+
+        let findClubTab = UINavigationController(rootViewController: ViewController()) //TODO: 모임 찾기 뷰로 연결
```

Comment:
```suggestion
    let findClubTab: UINavigationController = {
         let controller = UINavigationController(rootViewController: ViewController()
         controller.tabBarItem.image = UIImage(systemName: "person.3.fill")
         controller.title = "모임 찾기" 
     }()
    override func viewDidLoad() {
        super.viewDidLoad()
        
        view.backgroundColor = .white

```
이런식으로 클로저 내부에 구현해두면 어떤 역할을 하는지, 추후에 수정또한 간편해집니다.

Reply:
해당 부분도 클로저로 표현하면 좋겠네요! 감사합니다
57e7817f75244aa5459e55b2a4ea5c39ae822622 반영했습니다 😁